BotRoyaleUnity/Assets/TextMesh Pro/Examples & Extras/Materials.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
There's a bunch of added TextMesh Pro examples meta files. We shouldnt have any meta files in the repo without the actual files